src/main/java/com/seals/camplanner/event/controllers/EventController.java

```diff
@@ -16,9 +17,10 @@
     public List<Event> getEvents() {
         return eventService.findAll();
     }
+
     //TODO add treatment for the "location" parameter
     @PostMapping("/event")
-    public Event saveEvent(@RequestBody Event event) {
+    public Event saveEvent(@RequestBody EventDTO event) {
```

Comment:
This is a controversial subject, but I agree with [this](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)) source. It states that acronyms should be treated as any other word, ex:
* HTTP Request -> `httpRequest`
* Event DTO -> `eventDto`

What do you think about renaming `EventDTO` to `EventDto`?

Reply:
Good one. Thanks.